app/my-bets/page.tsx

```diff
@@ -116,7 +198,7 @@ const BetsListPanel = ({
     <TabPanel className="space-y-4">
       {isLoading && <LoadingBets />}
       {!isLoading &&
-        bets.length &&
+        bets.length > 0 &&
```

Comment:
Do we need this change? Feels redundant because `bets.length` should be truthy for any non-zero value and falsy for zero value.

Reply:
we need this to boolean otherwise it prints the value

Reply:
Ah I see, gotcha